src/consent-manager/container.tsx

```diff
@@ -147,10 +156,7 @@ const Container: React.FC<ContainerProps> = props => {
 
   const handleSave = () => {
     toggleDialog(false)
-    // If preferences haven't changed, don't reload the page as it's a disruptive experience for end-users
```

Comment:
This functionality (don't reload if nothing has changed) has been moved to consent manager builder.

Reply:
Edit - these comments were added before I saw @danielstjules 's review. 😄 Thanks!